development/wasm-filter-plugins.md

```diff
@@ -0,0 +1,162 @@
+# WASM Filter Plugins
+
+Fluent Bit currently supports integration of wasm plugins built as wasm/wasi objects for input and filter plugins only. The interface for the WASM filter plugins is currently under development but is functional.
+
+## Prerequisites
+
+### For Build fluent-bit
+
+For only execution of WASM input and filter plugins, there is no additional requirement.
```

Comment:
```suggestion
There are no additional requirements to execute WASM plugins.
```
No need to repeat filter and input here as when/if it changes we will have to remove it.